src/OpenTK/Math/Vector2.cs

```diff
@@ -779,6 +849,32 @@ public static void Transform(ref Vector2 vec, ref Quaternion quat, out Vector2 r
         }
 
         /// <summary>
+        /// Transform a Vector by the given Matrix using right-handed notation.
+        /// </summary>
+        /// <param name="vec">The vector to transform</param>
+        /// <param name="mat">The desired transformation</param>
+        /// <returns>The transformed vector</returns>
+        public static Vector2 operator *(Matrix3 mat, Vector2 vec)
```

Comment:
Could be a bit confusing allowing M3*V2. It's an implicit cast of Vector2 to a homogeneous Vector3 and back again. Ok when that's called out explicitly as a transform but it's not really a multiplication.